### ISSUE TITLE
Empty buttons in GlobalConfirmModal footer

### ISSUE THREAD

**Issue by funkycram** [humhub/humhub#6099]
### What steps will reproduce the problem?

Create a report on https://wave.webaim.org/report

### What is the expected result?

No error

### What do you get instead?

6 errors about the space stream filter form.
See bellow the full error.

### Additional info

The error comes from these 2 empty buttons: 
https://github.com/humhub/humhub/blob/master/protected/humhub/widgets/GlobalConfirmModal.php#L35

```php
 'footer' => '<button data-modal-cancel data-modal-close class="btn btn-default"></button><button data-modal-confirm data-modal-close class="btn btn-primary"></button>'
```

| Q                | A
| ---------------- | ---
| HumHub version   | 1.13.1

### Full error

**What It Means**

A button is empty or has no value text.

**Why It Matters**

When navigating to a button, descriptive text must be presented to screen reader users to indicate the function of the button.

**What To Do**

Place text content within the <button> element or give the <input> element a value attribute.

**The Algorithm... in English**

A <button> element is present that contains no text content (or alternative text), or an <input type="submit">, <input type="button">, or <input type="reset"> has an empty or missing value attribute.

**Comment by samuk** [comment]
Following issue. Accessibility is important to us.

**Comment by funkycram** [comment]
@luke- https://github.com/humhub/humhub/pull/6176
It pre-fills the buttons with default text.
Then, the JS will replace this text by the one in the clicked button HTML data.

**Comment by luke-** [comment]
@funkycram Thanks it's merged
